flask_app/get_data.py

```diff
@@ -32,13 +32,12 @@ def main():
 
 def date():
     with codecs.open(
-        "mocks/template.html", "r", "utf-8"
+        "Zastępstwa.html", "r", "utf-8"
```

Comment:
We already introduced this name on line 10. Maybe it's a good idea to create a constant with it.